_delphi_utils_python/delphi_utils/archive.py

```diff
@@ -75,10 +75,10 @@ def diff_export_csv(
 
     before_df = pd.read_csv(before_csv, dtype=export_csv_dtypes)
     before_df.set_index("geo_id", inplace=True)
-
+    before_df = before_df.round(7)
```

Comment:
Please note in the function docstring that this rounding is happening.

Reply:
added to both archive and export docstrings. also updated this to use `.round({"val": 7, "se": 7})` to be more explicit